### PR TITLE
Emergency fix for RHEL8 gating tests

### PR DESCRIPTION
**Pull request by edsantiago**
...because systemd on RHEL8 is too old, has no -P

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@giuseppe @vrothberg @containers/podman-maintainers PTAL

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

**Comment by Luap99** [comment]
Did you open a PR for main? We should not merge stuff that has no main PR otherwise we just regress with the next release again.

**Comment by edsantiago** [comment]
#17320 on main. That's what this PR started off as, before I decided to make it simple.

**Pull request merged**
